src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -86,6 +87,11 @@
 /* Between-page links and captions */
 nav.bd-links {
   font-size: var(--pst-sidebar-primary-font-size);
+  margin-right: -1rem;
```

Comment:
@12rambau do you remember why you styled `nav.bd-links` (the nav element containing the section navigation links) to have `margin-right: -1rem`? (Aside: can you tell me what the `bd-` prefix means?)

I can plainly see and understand why you set the negative -1rem margin for the in-page TOC in the right sidebar (it's so that the vertical notch that indicates the current section is aligned properly), but so far I haven't been able to figure out what the -1rem margin is doing for the left sidebar. (On the left sidebar, the current section notches are not aligned to any vertical line.)

Reply:
the `bd-` prefix is the bootstrap one that they use usually in their demo examples. it's a very old issue but for the release of version 1 we would like to get rid of them https://github.com/pydata/pydata-sphinx-theme/issues/37. 

About the `-1rem` I guess the only way to find out is to remove it 😄